### DISCUSSION TITLE
Importing and creating custom models

### DISCUSSION THREAD

**Discussion by alexzedim** [winkjs/wink-nlp#41] [New Features & Ideas]
Hi, @sanjayaksaxena!

As I understand it, you are a maintainer of this npm packet. I'd like to thank you, because it's very awesome and supports TS types out-of-the-box, instead of most of many other NLP packets.

I'd like to ask a question about custom importing models and converting other models (like BERT or GPT). For example, I'd like to take a custom [BERT model from this repositories](https://github.com/vlarine/transformers-ru). I am very interested of using Russian (Cyrillic) models, which supports this language.

I already downloaded and open `wink-eng-lite-model` and found that it's a bit different than other models.

Is there any option, like a converting model guideline for that?

**Comment by sanjayaksaxena** [comment]
Hi @AlexZeDim 

Delighted to know that you found winkNLP useful.

Thanks for your idea and we can possibly pursue it together if you would like to contribute.

Yes, our models are different — this is because typically the corpus needed to build the models have restrictive licenses making it difficult to release models under the MIT license.

The first step would be to build the tokenizer model for Russian. We have no exposure to Russian language. I believe that structurally it will be similar to languages based on Latin scripts. Subsequently we can plan its integration with BERT models that can be ported to work with winkNLP framework.

What do you think?
